service/prometheus/scrapeconfig_test.go

```diff
@@ -1,7 +1,6 @@
 package prometheus
 
 import (
-	"net/url"
```

Comment:
Everything below here is test refctoring and improvements. Because the configuration changed, most of the testing changed too.